lib/src/react_client/event_helpers.dart

```diff
@@ -760,44 +760,47 @@ SyntheticWheelEvent createSyntheticWheelEvent({
 }
 
 extension SyntheticEventTypeHelpers on SyntheticEvent {
+  bool _checkEventType(List<String> types) => hasProperty(this, 'type') && types.any((t) => this.type.contains(t));
+
   /// Whether the event instance has been removed from the ReactJS event pool.
   ///
   /// > See: [persist]
   @Deprecated('The modern event system does not use pooling. This always returns true.')
   bool get isPersistent => true;
 
   /// Uses Duck Typing to detect if the event instance is a [SyntheticClipboardEvent].
-  bool get isClipboardEvent => hasProperty(this, 'clipboardData');
+  bool get isClipboardEvent => hasProperty(this, 'clipboardData') || _checkEventType(['copy', 'paste', 'cut']);
```

Comment:
These all should be `const`s
```suggestion
  bool get isClipboardEvent => hasProperty(this, 'clipboardData') || _checkEventType(const ['copy', 'paste', 'cut']);
```